http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AzurePubsubHandler.scala

```diff
@@ -576,6 +573,8 @@ class AzurePubsubHandlerInterp[F[_]: Parallel](
           case Some(resp) =>
             resp.jobReport.status match {
               case WsmJobStatus.Succeeded =>
+                dbRef.inTransaction(clusterQuery.updateClusterStatus(runtime.id, RuntimeStatus.Deleted, ctx.now))
```

Comment:
So these are all `IO`s, which need to be chained together to form a single `IO` which is returned by this `match` statement. Otherwise they won't run.

So need something like:
```
dbRef.inTransaction(..) >> logger.info(..) >> deleteDiskAction
```

or

```
for {
  _ <- dbRef.inTransaction(..)
  _ <- logger.info(..)
  _ <- deleteDiskAction
yield ()
```

… which are both syntax for the same thing.

Reply:
ohhh yeah gotcha, I'll fix that

Reply:
for comprehensions for the win!